### ISSUE TITLE
Find recalls: Missing form label on 'Search website"

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#94]
URL: https://hcdev.openplus.ca/en
User Agent: Firefox 91.0.2
Language: Both

Issue Description: A form control does not have a corresponding label for 'Search website' in the top right hand corner of page.  Please refer to Canada.ca theme.

Steps to reproduce:
1. Open the 'Find recalls, advisories and safety alerts' page
2. Access any of the links on the page
3. Turn on WAVE 
RSAMS
![image](https://user-images.githubusercontent.com/68117281/131898733-934eb360-7604-43f8-907b-848049bec8f4.png)
Canada.ca
![image](https://user-images.githubusercontent.com/68117281/131898880-99de791e-eae0-465a-8a2c-adaaea4f93ec.png)

Expected results: The function and purpose of the form control must have a corresponding label to assist screen reader users

**Comment by ndolter** [comment]
Validated that the form label has been provided, however, the French page needs to be translated.
![image](https://user-images.githubusercontent.com/68117281/136271028-51b014af-458e-4af9-bb9d-fc0938456871.png)

**Comment by echouin** [comment]
@ndolter This is fixed.

**Comment by ndolter** [comment]
Validated that both Search fields are now translated.
![image](https://user-images.githubusercontent.com/68117281/137138888-b44869f8-15ea-405a-bb02-0a5d3cd1aacd.png)
